dts/bindings/xen,domu-vbd.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2024 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xen DomU configuration for virtual block device
+compatible: "xen,domu-vbd"
+
+include: base.yaml
+
+properties:
+  backend_domain_id:
+    type: int
+    default: 1
+    description: |
+      Designates a backend domain ID for the device.
```

Comment:
If we are doing it right (and I hope we are doing it right), this should be domain name. `domain_id` is no persistent and can change due to reboot or failed boot attempt.

libxl uses domain name and we should do the same